### DISCUSSION TITLE
Pay but not active

### DISCUSSION THREAD

**Discussion by Habibieproject** [community/community#147771] [Copilot Conversations]
Hello, i get a problem. 
I have pay GitHub Co-pilot Pro $10. But, until now, my account not yet change to GitHub Co-pilot pro in profile.. Any solution? 

Note : I pay on google play, one month ago I do same, and success.

**Comment by Akash1134** [comment]
Hi @Habibieproject  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Habibieproject** [reply]
Thanks for suggestion..
